### PR TITLE
[Update] delete once

### PR DESCRIPTION
**Pull request by Icemap**
Change infinity loop to once.

### CODE REVIEW

File: plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java [hunk 1]
```diff
@@ -38,13 +38,13 @@ public static void main(String[] args) throws InterruptedException {
         mysqlDataSource.setUser("root");
         mysqlDataSource.setPassword("");
 
-        while (true) {
-            batchDelete(mysqlDataSource);
-            TimeUnit.SECONDS.sleep(1);
+        Integer updateCount = -1;
+        while (updateCount != 0) {
```
Comment (Mini256) [inline plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:42]: Maybe `updateCount > 0`?

Comment (Icemap) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:42]: It will cause the loop can't run first.

Comment (Mini256) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:42]: OK, but if an exception causes return-1, the loop doesn't seem to end 🤔

Comment (Mini256) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:42]: maybe do ... while ... is better

Comment (Icemap) [reply plain-java-batch-delete/src/main/java/com/pingcap/bulkDelete/BatchDeleteExample.java:42]: 確かに。Good idea.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**